teacher.rb

```diff
@@ -0,0 +1,14 @@
+require './person'
+
+class Teacher < Person
+  attr_accessor :specialization
+
+  def initialize(age, specialization, name: 'Unknown', parent_permission: true)
+    super(age, name: name, parent_permission: parent_permission)
```

Comment:
- [ ] Same here kindly consider making the parameters shorter

```suggestion
    super(age, name, parent_permission)
```

Reply:
well resolved! ✔️